src/inverse-dynamics-balance-controller.cpp

```diff
@@ -362,39 +399,41 @@ void InverseDynamicsBalanceController::updateComOffset() {
 
 void InverseDynamicsBalanceController::removeRightFootContact(const double& transitionTime) {
   if (m_contactState == DOUBLE_SUPPORT) {
-    SEND_MSG("Remove right foot contact in " + toString(transitionTime) + " s", MSG_TYPE_INFO);
+    SEND_MSG("Remove right foot contact in " + toString(transitionTime) + " s", MSG_TYPE_ERROR);
```

Comment:
This doesn't look like an error msg

Reply:
Yes sorry it was for debug purpose I will change this